protected/models/Disorder.php

```diff
@@ -50,6 +51,8 @@ class Disorder extends BaseActiveRecordVersioned
     public static $SNOMED_DIABETES_TYPE_I_SET = array(46635009, 420868002);
     public static $SNOMED_DIABETES_TYPE_II_SET = array(44054006, 422014003);
 
+    public $is_ophthalmic;
```

Comment:
Local variables are used models/Patient, models/Document and a few other models. It seems like a common way to add variables to newly created objects without affecting the database.